tests/bin/setup-node.sh

```diff
@@ -9,23 +9,45 @@ set -eo pipefail
 apt-get install -y apt-transport-https
 
 # install docker
-apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 36A1D7869245C8950F966E92D8576A8BA88D21E9
-sh -c "echo deb https://get.docker.com/ubuntu docker main > /etc/apt/sources.list.d/docker.list"
-apt-get update && apt-get install -yq lxc-docker-1.5.0
+apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 \
+            --recv-keys 36A1D7869245C8950F966E92D8576A8BA88D21E9
+echo deb https://get.docker.com/ubuntu docker main > /etc/apt/sources.list.d/docker.list
+apt-get update
+apt-get install -yq lxc-docker-1.5.0
+
+# install extra extensions (AUFS)
+sudo apt-get -y install "linux-image-extra-$(uname -r)"
```

Comment:
@sgoings my suggestion to update the kernel is just because it takes less than 10 minutes and that solves the aufs issues. Is just temporal workaround. I don't have a reason to keep using 14.04 or even Ubuntu.